schemas/metaschema/DomainField.schema

```diff
@@ -1,5 +1,6 @@
 {
   type: Enum('string', 'number', 'boolean', 'function', 'object', 'symbol'),
+  class: { type: 'string' },
```

Comment:
Why do we need `class` here and this may give a conflict with keyword.

Reply:
I meant `DomainField` as in field in `domains.schema` object, so it's not the Domain field, it's a possible properties of Domain definition.

I've added `class` because it's used in `domain.schema` (https://github.com/metarhia/metaschema/blob/master/schemas/metaschema/domains.schema#L18)